CMakeLists.txt

```diff
@@ -114,6 +114,7 @@ add_executable(test_algo ${PERF_SOURCES})
 target_link_libraries(test_algo
   algos
   "${binary_dir}/lib/libopencv_ts.a"
+  "${binary_dir}/lib/libopencv_imgcodecs.so"
```

Comment:
Add `"${binary_dir}/install/lib/libopencv_imgcodecs.so"` to `target_link_libraries(algos` instead:
```patch
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -92,6 +92,7 @@ target_include_directories(algos PRIVATE
 target_link_libraries(algos
   "${binary_dir}/install/lib/libopencv_core.so"
   "${binary_dir}/install/lib/libopencv_imgproc.so"
+  "${binary_dir}/install/lib/libopencv_imgcodecs.so"
 )
```